### PR TITLE
Bug 1376933 Added context to QoS topic

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1376933

It's a old one, so I think the required info was there. Will verify in the BZ.

### CODE REVIEW

**Comment by TheDiemer** [comment]
LGTM @bfallonf

**Comment by bfallonf** [comment]
Thanks @TheDiemer ! I'll put to review.

@openshift/team-documentation PTAL

File: dev_guide/compute_resources.adoc [hunk 1]
```diff
@@ -251,8 +251,9 @@ endif::[]
 [[quality-of-service-tiers]]
 === Quality of Service Tiers
 
-A compute resource is classified with a _quality of service_ (QoS) based on the
-specified request and limit value.
+When created, a compute resource is classified with a _quality of service_
+(QoS). There are three tiers, and each are based on the request and limit value
```
Comment (kalexand-rh) [inline dev_guide/compute_resources.adoc]: s/are/is

**Review by kalexand-rh — COMMENTED** [review]
One change, otherwise LGTM

**Comment by bfallonf** [comment]
Thanks @kalexand-rh . Fixed. I'll merge.

**Pull request merged**